### PR TITLE
Added support to colored stat nicknames

### PR DESCRIPTION
**Pull request by nauar**
Added support for colored stat nicknames.

IRC:

image

DISCORD:

image

### CODE REVIEW

**Comment by nauar** [comment]
BTW, the name of the branch does not correspond to the feature implemented :laughing:  My bad

**Review by Seekfried — CHANGES_REQUESTED** [review]
we don't need the old def get_statsname(self, id) anymore, also is the command_testcolors still needed?

File: friedybot.py [hunk 1]
```diff
@@ -42,6 +43,126 @@ def __get_active_player_entries(self, player) -> PickupEntries:
...
+        elif h < 150:
+            return 2
+        elif h < 200:
+            return 5
+        elif h < 270:
+            return 4
+        elif h < 330:
+            return 6
+        else:
+            return 1
```
Comment (Seekfried) [inline friedybot.py:79]: maybe just my opinion, to make it a little more compact:
```suggestion
        if h < 36:
            return 1
        elif h < 80:
            return 3
        elif h < 150:
            return 2
        elif h < 200:
            return 5
        elif h < 270:
            return 4
        elif h < 330:
            return 6
        else:
            return 1
```
color_thresholds =[(36,1), (80,3), (150,2),(200,5),(270,4),(330,6)]

for theshold, value in color_thresholds:
    if h < threshold:
        return value

return 1

**Comment by nauar** [comment]
> we don't need the old def get_statsname(self, id) anymore, also is the command_testcolors still needed?

No, I'm sorry. Removing these methods.

**Pull request merged**